packages/flutter/lib/src/widgets/image_filter.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'framework.dart';
+
+/// Applies an [ImageFilter] to its child.
+@immutable
+class ImageFiltered extends SingleChildRenderObjectWidget {
+  /// Creates a widget that applies an [ImageFilter] to its child.
+  ///
+  /// The [imageFilter] must not be null.
+  const ImageFiltered({@required this.imageFilter, Widget child, Key key})
+      : assert(imageFilter != null),
+        super(key: key, child: child);
```

Comment:
trivial nit: flutter style here is:

```dart
  const ImageFiltered({
    Key key,
    @required this.imageFilter,
    Widget child,
  }) : assert(imageFilter != null),
￼       super(key: key, child: child);
```

Reply:
Same annoying A.S. broken formatting. Fixed...